src/main/java/com/bankbazaar/test/controller/TestController.java

```diff
@@ -0,0 +1,30 @@
+package com.bankbazaar.test.controller;
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.bankbazaar.test.model.TestModel1;
+import com.bankbazaar.test.model.TestModel2;
+
+@RestController
+public class TestController {
+    @GetMapping("/")
+    public String index()
+    {
+        return "Hello World";
+    }
+    @GetMapping("/API1")
+    public String getAPi1()
+    {
+        TestModel1 test1 = new TestModel1();
+        return test1.getData();
+    }
+
+    @GetMapping("/API2")
+    public String getAPi2()
```

Comment:
Add a queryParam to pass data from the user

Reply:
@RequestParam used